### PR TITLE
Beautified script, added sudo prompt and exit message

### PR DESCRIPTION
**Pull request by sachinkamath**
The script now prompts for sudo access before running the program. If sudo was explicitly invoked by the user, then no prompt is made.

Also fixed indentations and enhanced readability. @delta24 What do you think?

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -119,39 +130,39 @@ scriptdir=$(dirname $0)
...
+        print_info " Fetching repository and starting installation process.."
+        git clone -b develop $dev; cd owtf/
+        Install
+        break
+      ;;
+      "Quit")
+        print_info " If you wish to install OWTF later, please run this script again.."
+        break
+      ;;
+      *) echo -e "Invalid option. Try again."; continue ;;
```
Comment (delta24) [inline bootstrap.sh]: Format the echo the next line with indentation

File: bootstrap.sh [hunk 2]
```diff
@@ -59,6 +59,11 @@ BWhite=${Bold}$(tput setaf 7)
 print_line() {
   printf "%$(tput cols)s\n"|tr ' ' '*'
 }
+
+prompt_sudo() {
+	[ "$UID" -eq 0 ] || exec sudo bash "$0" "$@"
```
Comment (delta24) [inline bootstrap.sh]: Indentation with 2 spaces only.

**Comment by sachinkamath** [comment]
Ouch, my bad :stuck_out_tongue_closed_eyes: Fixed @delta24

**Comment by sachinkamath** [comment]
Ouch, that indentation mistake was stupid. Fixed @delta24

File: bootstrap.sh [hunk 3]
```diff
@@ -119,39 +130,39 @@ scriptdir=$(dirname $0)
...
+        print_info " Fetching repository and starting installation process.."
+        git clone -b develop $dev; cd owtf/
+        Install
+        break
+      ;;
+      "Quit")
+        print_info " If you wish to install OWTF later, please run this script again.."
+        break
+      ;;
+      *) echo -e " Invalid option. Try again."; continue ;;
```
Comment (delta24) [inline bootstrap.sh]: This `echo` statement should not be inlined.

**Pull request merged**